plugins/modules/zos_copy.py

```diff
@@ -2182,6 +2260,7 @@ def run_module(module, arg_def):
     dest = module.params.get('dest')
     remote_src = module.params.get('remote_src')
     is_binary = module.params.get('is_binary')
+    executable = module.params.get('executable')
```

Comment:
I'm curious about the choice of naming both the variable and the interface just plain "executable" vs "is_executable"?

Reply:
Can it be replaced for is_executable?